add_local_user causing a terraform crash

### Terraform Version

PS C:\Users\A-Boyd\Documents\Dev\Terraform\Projects\ACI> terraform -v
Terraform v1.8.2
on windows_amd64
+ provider registry.terraform.io/ciscodevnet/aci v2.14.0

### APIC version and APIC Platform

APIC-Version 5.2 (8g), APIC-SERVER-L3

### Affected Resource(s)

aci_local_user resource

### Terraform Configuration Files

<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key: https://keybase.io/hashicorp
```

### Panic Output

aci_local_user.example: Creating...
aci_local_user.example: Still creating... [10s elapsed]
╷
│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-aci_v2.14.0.exe plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x132aeb5]

goroutine 24 [running]:
github.com/CiscoDevNet/terraform-provider-aci/v2/aci.tryLogin(0xc000040330?, 0xc0000ea7e0)
        github.com/CiscoDevNet/terraform-provider-aci/v2/aci/resource_aci_aaauser.go:184 +0x275
github.com/CiscoDevNet/terraform-provider-aci/v2/aci.resourceAciLocalUserCreate({0x1f4e880, 0xc000201030}, 0xc0005b8d80, {0x1d06980?, 0xc0000ea7e0?})
        github.com/CiscoDevNet/terraform-provider-aci/v2/aci/resource_aci_aaauser.go:318 +0x56d
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc000326000, {0x1f4e7d8, 0xc000538f60}, 0xd?, {0x1d06980, 0xc0000ea7e0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.33.0/helper/schema/resource.go:778 +0x11b
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc000326000, {0x1f4e7d8, 0xc000538f60}, 0xc00059a820, 0xc0005b8c00, {0x1d06980, 0xc0000ea7e0})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.33.0/helper/schema/resource.go:909 +0xa89
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc000008270, {0x1f4e7d8?, 0xc000538990?}, 0xc00058b040)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.33.0/helper/schema/grpc_provider.go:1078 +0xdbc
github.com/hashicorp/terraform-plugin-mux/tf5to6server.v5tov6Server.ApplyResourceChange({{0x1f55a80?, 0xc000008270?}}, {0x1f4e7d8, 0xc000538990}, 0x0?)
        github.com/hashicorp/terraform-plugin-mux@v0.15.0/tf5to6server/tf5to6server.go:47 +0x54
github.com/hashicorp/terraform-plugin-mux/tf6muxserver.(*muxServer).ApplyResourceChange(0x1f4e810?, {0x1f4e7d8?, 0xc000538690?}, 0xc00058aff0)
        github.com/hashicorp/terraform-plugin-mux@v0.15.0/tf6muxserver/mux_server_ApplyResourceChange.go:36 +0x193
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ApplyResourceChange(0xc000258140, {0x1f4e7d8?, 0xc00068bc80?}, 0xc000200770)
        github.com/hashicorp/terraform-plugin-go@v0.22.1/tfprotov6/tf6server/server.go:846 +0x3d0
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ApplyResourceChange_Handler({0x1cd3900?, 0xc000258140}, {0x1f4e7d8, 0xc00068bc80}, 0xc0005b8400, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.22.1/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:518 +0x169
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00012da00, {0x1f4e7d8, 0xc00068bbf0}, {0x1f545a0, 0xc000059520}, 0xc0000f9560, 0xc000516b70, 0x2955178, 0x0)
        google.golang.org/grpc@v1.62.1/server.go:1386 +0xe23
google.golang.org/grpc.(*Server).handleStream(0xc00012da00, {0x1f545a0, 0xc000059520}, 0xc0000f9560)
        google.golang.org/grpc@v1.62.1/server.go:1797 +0x100c
google.golang.org/grpc.(*Server).serveStreams.func2.1()
        google.golang.org/grpc@v1.62.1/server.go:1027 +0x8b
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 37
        google.golang.org/grpc@v1.62.1/server.go:1038 +0x135

Error: The terraform-provider-aci_v2.14.0.exe plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

### Expected Behavior

Local user should be created in APIC

### Actual Behaviour

Terraform crash and no user added

### Steps to Reproduce

1. `terraform apply` (crash occurs on apply)

### Important Factoids


### References

--- a/aci/resource_aci_aaauser.go
+++ b/aci/resource_aci_aaauser.go
@@ -182,7 +182,7 @@ func tryLogin(d *schema.ResourceData, aciClient *client.Client) bool {
 	client.SkipLoggingPayload(false)(aciClient)
 
 	if err != nil {
-		log.Printf("[DEBUG] Authentication failed for local user: '%s'. ERROR: %s", d.Get("name").(string), err)
+		log.Printf("[DEBUG] Authentication failed for local user: '%s'. ERROR: %v", d.Get("name").(string), err)
 		return false
 	} else if resp.StatusCode != 200 {
 		log.Printf("[DEBUG] Authentication failed for local user: '%s'.", d.Get("name").(string))

--- a/aci/resource_aci_aaauser.go
+++ b/aci/resource_aci_aaauser.go
@@ -177,11 +177,14 @@ func tryLogin(d *schema.ResourceData, aciClient *client.Client) bool {
 		return false
 	}
 
-	_, resp, _ := aciClient.Do(req)
+	_, resp, err := aciClient.Do(req)
 
 	client.SkipLoggingPayload(false)(aciClient)
 
-	if resp.StatusCode != 200 {
+	if err != nil {
+		log.Printf("[DEBUG] Authentication failed for local user: '%s'. ERROR: %s", d.Get("name").(string), err)
+		return false
+	} else if resp.StatusCode != 200 {
 		log.Printf("[DEBUG] Authentication failed for local user: '%s'.", d.Get("name").(string))
 		return false
 	}

